src/components/WebexSettings/Tabs.jsx

```diff
@@ -29,49 +30,51 @@ export default function Tabs({
   style,
   tabs,
   tabIndex,
+  autoFocus,
 }) {
   const {content} = tabs.find((tab) => tab.key === selected);
   const [cssClasses, sc] = webexComponentClasses('tabs', className);
   const selectedTabIndex = (tabs || []).findIndex((tab) => tab.key === selected);
   const ref = useRef();
+  const [focus, setFocus] = useState(false);
```

Comment:
this state seems to be a flag indicating that the tab has changed since the initial rendering.

Let's rename it for better semantics:
```js
const [tabChangedWithKey, setTabChangedWithKey] = useState(false);